developer-docs/api.md

```diff
@@ -0,0 +1,109 @@
+# ACE API
+
+## Overview
+ACE provides 2 APIs to enable the execution or Tasks and Catalog compilation for a remote target.
+
+## API Endpoints
+Each API endpoint accepts a request as described below. The request body must be a JSON object.
+
+### POST /run_task
+- `connection-info`: [RSAPI Transport Object](#rsapi-transport-object), *required* - Target information to run task on.
+- `task`: [Task Object](#task-object), *required* - Task to run on target.
+- `parameters`: Object, *optional* - JSON formatted parameters to be provided to task.
+
+For example, the following runs the 'echo' task on linux_target.net:
+```
+{
+  "connection-info": {
```

Comment:
I think we should stay as close as possible to bolt-server and also call this `target`.

Reply:
If you look at where this is defined, it does not touch the JSONSchema for a task, which stays true to Bolt. Changing this to `connection-info` makes it clear that this does not contain the same information as `target`